bin/snare

```diff
@@ -0,0 +1,209 @@
+                repo.git.reset('--hard')
+                repo.heads.master.checkout()
+                repo.git.clean('-xdf')
+                repo.remotes.origin.pull()
+                pip.main(['install', '-r', 'requirements.txt'])
+                os.execv(sys.executable, [sys.executable, __file__] + sys.argv[1:])
+                return
+            else:
+                print('you are running the latest version')
+            time.sleep(timeout)
+
+
+async def check_tanner():
+    vm = snare_helpers.VersionManager()
+    async with aiohttp.ClientSession() as client:
+        req_url = 'http://{}:8090/version'.format(args.tanner)
+        try:
+            resp = await client.get(req_url)
+            result = await resp.json()
+            version = result["version"]
+            vm.check_compatibility(version)
+        except aiohttp.ClientOSError:
+            print("Can't connect to tanner host {}".format(req_url))
+            exit(1)
+        else:
+            await resp.release()
+
+if __name__ == '__main__':
+    print(r"""
+   _____ _   _____    ____  ______
+  / ___// | / /   |  / __ \/ ____/
+  \__ \/  |/ / /| | / /_/ / __/
+ ___/ / /|  / ___ |/ _, _/ /___
+/____/_/ |_/_/  |_/_/ |_/_____/
+
+    """)
+    parser = argparse.ArgumentParser()
+    page_group = parser.add_mutually_exclusive_group(required=True)
+    page_group.add_argument("--page-dir", help="name of the folder to be served")
+    page_group.add_argument("--list-pages", help="list available pages", action='store_true')
+    parser.add_argument("--index-page", help="file name of the index page", default='index.html')
+    parser.add_argument("--port", help="port to listen on", default='8080')
+    parser.add_argument("--interface", help="interface to bind to")
+    parser.add_argument("--host-ip", help="host ip to bind to", default='localhost')
+    parser.add_argument("--debug", help="run web server in debug mode", default=False)
+    parser.add_argument("--tanner", help="ip of the tanner service", default='tanner.mushmush.org')
+    parser.add_argument("--skip-check-version", help="skip check for update", action='store_true')
+    parser.add_argument("--slurp-enabled", help="enable nsq logging", action='store_true')
+    parser.add_argument("--slurp-host", help="nsq logging host", default='slurp.mushmush.org')
+    parser.add_argument("--slurp-auth", help="nsq logging auth", default='slurp')
+    parser.add_argument("--config", help="snare config file", default='snare.cfg')
+    parser.add_argument("--auto-update", help="auto update SNARE if new version available ", default=True)
+    parser.add_argument("--update-timeout", help="update snare every timeout ", default='24H')
+    parser.add_argument("--server-header", help="set server-header", default='nignx/1.3.8')
+    parser.add_argument("--no-dorks", help="disable the use of dorks", type=str_to_bool,  default=True)
```

Comment:
I have changed this line because previously it was not a user-input argument, now i have used str_to_bool type to get input whether to disable dorks or not (True or False) and process it accordingly.